cddl_gen/cddl_gen.py

```diff
@@ -19,11 +19,19 @@
 from json import loads as json_load, dumps as json_dump
 from io import BytesIO
 from subprocess import Popen, PIPE
+from pathlib import Path
 import sys
 
 indentation = "\t"
 newl_ind = "\n" + indentation
 
+P_SCRIPT = Path(__file__).absolute().parents[0]
+P_REPO_ROOT = Path(__file__).absolute().parents[1]
+VERSION_path = Path(P_SCRIPT, "VERSION")
+
+with open(VERSION_path, 'r') as f:
+    __version__ = f.read()
```

Comment:
```suggestion
    __version__ = f.read()).strip()
```
For safety's sake, in case this brings in a newline character from a future modification.